src/ows/src/main/java/org/geoserver/ows/Dispatcher.java

```diff
@@ -1151,6 +1151,10 @@ Service findService(String id, String ver, String namespace) throws ServiceExcep
         if (matches.size() > 1) {
             List vmatches = new ArrayList(matches);
 
+            if (isCiteCompliant() && version == null && "WFS".equals(id)) {
```

Comment:
The problem with this implementation is that CITE tests run with the `ows` endpoint, for which `isCiteCompliant()` is `false`. You can find the services and examine them for this setting, but you can never access their content because `gs-wfs` in which `WFSInfo` is defined depends on `gs-ows` and is out of scope.

I think an alternate solution is required, perhaps disabling WFS 2.0.0 either in code or in the `wfs.xml` settings. I do not think the latter will be honoured. Perhaps we need something to disable WFS 2.0.0 when WFS CITE compliance is enabled?